lib/utils/prometheus.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2015-2021 Gravitational, Inc.
```

Comment:
Nit: Just do `2021`, we only usually write the year the file was created/updated in.